### PR TITLE
Create IBM.ibm-commonui-operator.release-1.5.yaml

### PR DESCRIPTION
**Pull request by siddarthpatel**
Enable prow for `release-1.5` commonui operator branch

### CODE REVIEW

**Review by ashank07 — APPROVED** [review]

File: prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.5.yaml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    always_run: true
+    path_alias: github.com/IBM/ibm-commonui-operator
+    rerun_command: /test build ibm-commonui-operator
+    spec:
+      containers:
+      - command:
+        - entrypoint
+        - make
+        - build
+        image: quay.io/multicloudlab/build-tool:v20200817-b1f2b4c05
```
Comment (horis233) [inline prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.5.yaml:16]: @siddarthpatel Any reason why we don't use the image in the master branch?

Comment (horis233) [reply prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.5.yaml:16]: The image used for the master branch includes `go version  1.15.7` can avoid some vulnerability issues.

Comment (siddarthpatel) [reply prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.5.yaml]: That was a mistake by me. I update to use the images in master branch

File: prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.5.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ presubmits:
   - name: build_ibm-commonui-operator
     branches:
     - ^release-1.5$
-    decorate: true
+    decorate: truef
```
Comment (horis233) [inline prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.5.yaml:6]: an extra `f` here

**Pull request merged**